ppa/archive/views.py

```diff
@@ -247,7 +253,56 @@ class DigitizedWorkDetailView(DetailView):
     model = DigitizedWork
     slug_field = 'source_id'
     slug_url_kwarg = 'source_id'
+    paginate_by = 50
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        # pull in the query if it exists to use
+        query = self.request.GET.get('query', '')
+
+        solr_pageq = None
+        if query:
+            context['query'] = query
+            digwork = context['object']
+            solr_q = 'text:(%s) AND srcid:("%s") AND item_type:(page)' \
+                % (query, digwork.source_id)
+            solr_opts = {
+                'q': solr_q,
+                'sort': 'order asc',
+                'fl': '*',
```

Comment:
Let's limit the fields to what we actually need for display (maybe just id right now?); in particular I'd like to not include `content`, since that makes the result so much smaller.